tests/infrared/13/infrared-openstack.sh

```diff
@@ -67,7 +67,6 @@ infrared tripleo-overcloud \
     --deploy yes \
     --public-network yes \
     --public-subnet default_subnet \
-    --ntp-server "${NTP_SERVER}" \
```

Comment:
The NTP_SERVER param is needed because public NTP servers were inaccessible from inside the VPN.

Reply:
this is one of the changes I was referring to that needs to be added. In the case of 13 the DnsServer parameter was passed in an environment file.

This is just a dump of the data so far and I know a bunch of things that need cleaning up.